polardrift_toolwin.cpp

```diff
@@ -190,6 +193,13 @@ wxCAPTION | wxCLOSE_BOX | wxMINIMIZE_BOX | wxSYSTEM_MENU | wxTAB_TRAVERSAL | wxF
     m_hemiChoice->SetToolTip(_("Select your hemisphere"));
     gbSizer->Add(m_hemiChoice, wxGBPosition(gridRow, 1), wxGBSpan(1, 1), wxALL, 5);
 
+    // Next row of grid
+    gridRow++;
+    m_mirrorCheck = new wxCheckBox(this, ID_MIRROR, _("Mirror image"));
+    gbSizer->Add(m_mirrorCheck, wxGBPosition(gridRow, 0), wxGBSpan(1, 1), wxALL | wxALIGN_BOTTOM, 5);
+    m_mirrorCheck->SetValue(m_mirror==-1?true:false);
```

Comment:
this is fine, but equivalent to
```suggestion
    m_mirrorCheck->SetValue(m_mirror == -1);
```